### DISCUSSION TITLE
Request for information

### DISCUSSION THREAD

**Discussion by D0nn1X** [rxhanson/Rectangle#1006] [General]
Dear Ryan, 

 

Hope you are well, 

I am contacting you regarding the Rectangle app, I wanted to know details about the app and the security aspect and I want to be sure that I can provide the app on production.

 

Could you please give me more detail about it ?

 

Data shared between the host and your servers

Telemetry 

Actions implemented to guaranty end user’s cybersecurity

 

Thank you in advance 

BR,

**Comment by rxhanson** [comment]
Here is the privacy policy: https://rectangleapp.com/terms

Rectangle will only reach out to https://rectangleapp.com for update information if enabled/requested and will download updates from the releases on this Github repository. I don’t track users or use telemetry. 

The app is of course notarized, and I don’t let in any code that would be a security risk.
